client/src/components/Lyrics/LyricsStyle.jsx

```diff
@@ -40,11 +38,12 @@ export const Paragraph = styled.p`
   text-align: right;
   font-family: "Aclonica";
   font-style: normal;
-  /* line-height: 150%; */
   font-size: 0.7rem;
-  margin-bottom: 1.5rem;
   color: black;
   width: 16ch;
-  height: 5ch;
   white-space: pre-line;
 `;
+
+export const WrapperDiv = styled.div`
+  height: 100%;
```

Comment:
if this is the only style applied, add it inline in jsx, no need to create a styled component for this